src/sdkV2.ts

```diff
@@ -93,7 +93,30 @@ export default class CegaEvmSDKV2 {
     });
   }
 
-  async bulkOpenVaultDepositsDcs(
+  async bulkSettleVaultsDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkSettleDCSVaults(vaultAddresses);
+  }
+
+  async bulkStartTradesDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkStartDCSTrades(vaultAddresses);
+  }
+
+  async submitDispute(vaultAddress: EvmAddress): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.submitDispute(vaultAddress);
+  }
```

Comment:
We already have these functions, think you need to update your branch to match main
https://github.com/cega-fi/cega-sdk-evm/blob/main/src/sdkV2.ts#L107-L125

Reply:
also not sure if this is just github doing a weird diff

Reply:
oh yeah it's github re-arranging the file (you can see that it removed it from lower on the page)